library/Icingadb/Model/Behavior/Binary.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/* Icinga DB Web | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Model\Behavior;
+
+use ipl\Orm\Contract\PropertyBehavior;
+use ipl\Orm\Contract\QueryAwareBehavior;
+use ipl\Orm\Contract\RewriteFilterBehavior;
+use ipl\Orm\Query;
+use ipl\Sql\Adapter\Pgsql;
+use ipl\Stdlib\Filter\Condition;
+
+use function ipl\Stdlib\get_php_type;
+
+/**
+ * Support hex filters for binary columns and PHP resource (in) / bytea hex format (out) transformation for PostgreSQL
+ */
+class Binary extends PropertyBehavior implements QueryAwareBehavior, RewriteFilterBehavior
+{
+    /**
+     * Properties for {@link rewriteCondition()} to support hex filters for each adapter
+     *
+     * Set in {@link setQuery()} from the {@link $properties} array because the latter is reset for adapters other than
+     * {@link Pgsql}, so {@link fromDb()} and {@link toDb()} don't run for them.
+     *
+     * @var array
+     */
+    protected $rewriteSubjects;
+
+    /**
+     * {@inheritDoc}
```

Comment:
I thought we don't use this anymore. Just define your `@throws`. Any sophisticated doc parser can merge this with the base doc block. (e.g. PHPStorm)

Reply:
Cool!